src/app/personside/infotabs/ytelser/sykepenger/utils/sykepengerUtils.ts

```diff
@@ -0,0 +1,13 @@
+import { Sykmelding } from '../../../../../../models/ytelse/sykepenger';
+import { Periode } from '../../../../../../models/periode';
+import { getNewestDate, getOldestDate } from '../../../../../../utils/dateUtils';
+
+export function getSykemeldingPeriode(sykmeldinger: Sykmelding[]) {
```

Comment:
Trenger denne en egen fil? Kommer den til å bli brukt et annet sted?

Reply:
Sier han som elsker filer hehe 😃

Reply:
Men tenker Sykemelding.tsx ikke er alt for stor per nå, og den ligger såppas tett knyttet med resten av logikken i den filen

Reply:
Ja, blir vel en smakssak :-P Har ofte prøvd å trekke ut slik logikk som utleder info fra data i egne filer som jeg også skriver litt nøyere tester på, synes det har funka greit..